frontend/rust-lib/flowy-error/src/impl_from/cloud.rs

```diff
@@ -21,6 +21,11 @@ impl From<AppResponseError> for FlowyError {
       AppErrorCode::NotEnoughPermissions => ErrorCode::NotEnoughPermissions,
       AppErrorCode::NetworkError => ErrorCode::HttpError,
       AppErrorCode::PayloadTooLarge => ErrorCode::CloudRequestPayloadTooLarge,
+      AppErrorCode::UserUnAuthorized => match &*error.message {
```

Comment:
It can be a temporary solution. Refactor it and use code instead of message as the match condition in the next version.

Reply:
Got it. Doing that in the next release.